### PR TITLE
fix: rename envelopeType to signatureFormat

### PR DESCRIPTION
**Pull request by JeyJeyGao**
Signed-off-by: Junjie Gao <junjiegao@microsoft.com>

### CODE REVIEW

**Comment by rgnote** [comment]
The targeted users of Notation CLI may not know what is an envelope. SignatureFormat is more reasonable for a dummy user.

File: config/config.go [hunk 1]
```diff
@@ -23,8 +23,8 @@ type Config struct {
 	InsecureRegistries       []string                 `json:"insecureRegistries"`
 	CredentialsStore         string                   `json:"credsStore,omitempty"`
 	CredentialHelpers        map[string]string        `json:"credHelpers,omitempty"`
-	// EnvelopeType defines the envelope type for signing
-	EnvelopeType string `json:"envelopeType,omitempty"`
+	// SignatureFormat defines the envelope type for signing
```
Comment (patrickzheng200) [inline config/config.go:26]:
```suggestion
	// SignatureFormat defines the signature envelope type for signing
```

File: config/config.go [hunk 2]
```diff
@@ -23,8 +23,8 @@ type Config struct {
 	InsecureRegistries       []string                 `json:"insecureRegistries"`
 	CredentialsStore         string                   `json:"credsStore,omitempty"`
 	CredentialHelpers        map[string]string        `json:"credHelpers,omitempty"`
```
Comment (priteshbandi) [inline config/config.go:25]: Do we need `CredentialsStore`, `VerificationCertificates` and `CredentialHelpers` ?

**Review by priteshbandi — APPROVED** [review]
LGTM, Added one comment for future cleanup.

**Review by rgnote — APPROVED** [review]

File: config/config.go (hunk 2 above)
Comment (JeyJeyGao) [reply config/config.go:25]: CredentialsStore and CredentialHelpers are using for login/logout command.
VerificationCertificates will be no longer used after TS/TP.

File: config/config.go (hunk 1 above)
Comment (JeyJeyGao) [reply config/config.go]: Done.

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**